ConsentViewController/Classes/GDPRAction.swift

```diff
@@ -33,7 +33,7 @@ import Foundation
 @objcMembers public class GDPRAction: NSObject {
     public let type: GDPRActionType
     public let id: String?
-    public let consentLanguage: String?
+    public var consentLanguage: String?
```

Comment:
@vilasma does `consentLanguage` need to be a `var`? Are we reassigning it at some point?

Reply:
Yes @andresilveirah, In native message case we are assigning the `Locale` value to `consentLanguage`.

Reply:
I have updated consentLanguage property type to let